src/app/code/community/Meanbee/OSD/Helper/Config.php

```diff
@@ -36,6 +37,18 @@ public function isEnabled($store = null)
     }
 
     /**
+     * Check if breadcrumbs are enabled in the organisational data output.
+     *
+     * @param Mage_Core_Model_Store|int|null $store
+     *
+     * @return bool
+     */
+    public function isBreadcrumbsEnabled($store = null)
+    {
+        return Mage::getStoreConfigFlag(static::XML_PATH_MODULE_BREADCRUMBS_ENABLED, $store);
```

Comment:
Should the current enabled flag be a master switch for the entire module?  At the moment, the module is defaulted to off but breadcrumbs to enabled.